### ISSUE TITLE
Unwanted behaviour of the reference strip

### ISSUE THREAD

**Issue by erikradisch** [openseadragon/openseadragon#1825]
Hi all,

I think I spoted an unwanted behaviour of the reference strip. I am Integrating Openseadragon into an Project Website. Here. I have around 1/3 of the Page to do so. As I have more than one picture to show, I thought, It might by nice to use the reference strip to allow the user to switch through the pictures. The problem is however, that my preview pictures are ridiculously small (see attached picture, those litle whtie dots within the black box are the preview pictures...). I think, it might be connected to the fact, that I use only one third of the page? Is there a possibility to tell the system how big the preview pictures should be?

![test](https://user-images.githubusercontent.com/34335811/82951793-23132200-9fa8-11ea-9fe1-69a6bb1631b4.png)
This is the written code, I use:

```
	    var viewer =  $wnd.OpenSeadragon({
	        element: where,
	        showRotationControl: true,
	        showFlipControl: true,
	        sequenceMode:       true,
	        maxZoomLevel: 100,
			prefixUrl: "scripts/openseadragon-bin-2.4.2/images/",
			tileSources: source,
			showReferenceStrip: true,
			referenceStripSizeRatio: 0.5,
			referenceStripHeight:100,
			referenceStripWidth:100
		});
		viewer.addHandler("pre-full-page", function (data) {
				data.preventDefaultAction=true;
				openFullscreen();
		});
		function openFullscreen() {
  			if (where.requestFullscreen) {
    			where.requestFullscreen();
  			} else if (where.mozRequestFullScreen) { 
    			where.mozRequestFullScreen();
  			} else if (where.webkitRequestFullscreen) { 
    			where.webkitRequestFullscreen();
  			} else if (where.msRequestFullscreen) {
    			where.msRequestFullscreen();
  			}
		}
```

**Comment by iangilman** [comment]
Sure enough, referenceStripWidth and referenceStripHeight seem to only affect the reference strip itself. The reason the reference strip is so narrow is because of the `referenceStripWidth: 100`. Looks like you can change the size of the images themselves by using `referenceStripSizeRatio`. Something like 0.1 or smaller seems to do the trick.

Unfortunately the reference strip hasn't gotten a lot of love, but it might be nice to take a look at improving the parameters for it (or at least better documenting what the current parameters do).

**Comment by iangilman** [comment]
A little more discussion about these parameters here: https://github.com/openseadragon/openseadragon/issues/1826

**Comment by msalsbery** [comment]
@iangilman Yes, ReferenceStrip needs love...taking it on next, as I've had to dive into it to finish the current MouseTracker/events patch to fix some naughtiness.

Assigning to self...

**Comment by iangilman** [comment]
Awesome!
